.travis.yml

```diff
@@ -26,8 +28,7 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower phantomjs-prebuilt
-  - bower --version
+  - npm install -g phantomjs-prebuilt
```

Comment:
We should convert this to use chrome rather than phantom. Can you make that change?

Reply:
I notice below we're configuring testem to run Chrome over phantom. We should definitely update this in conjunction with that change.